app/javascript/pages/components/Browser.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { history, store } from '../store';
```

Comment:
Since we're balancing the current purpose of Redux/whether we should remove it or buy more into it, I'm curious if using the history in the Redux store may be more helpful than maintaining a separate history that is exported from the same place.

Reply:
I thought that this would just use the same store when I did this way since I was importing the store file, but if not then I need to dig deeper into figuring out to wire it up. If you have a good link on this please send my way.

Reply:
This is actually one of the things that I'm still untangling. In the store file, we've got exports for a `createrBrowserHistory` object and a `createStore` object, the latter of which looks like it's pulling from the former. But maybe `history` is being used as both a variable and a parameter name?